backend/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from fastapi.testclient import TestClient
+import sys
+import os
+import json
+from pathlib import Path
+
+# Add the parent directory to Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I know this is a common workaround (they spammed this during my PEY) but it's actually quite a messy workaround. I think a fix would be to just add an `__init__.py` to the `app/` directory in the backend so that python can detect it as a parent package.